G/GettextTiny/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+using BinaryBuilder
+
+# Collection of sources required to build Gettext
+name = "GettextTiny"
+version = v"0.3.2"
+sources = [
+    ArchiveSource("https://ftp.barfooze.de/pub/sabotage/tarballs/gettext-tiny-$(version).tar.xz",
+                  "a9a72cfa21853f7d249592a3c6f6d36f5117028e24573d092f9184ab72bbe187"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gettext-tiny*/
+
+export LDFLAGS="-L${libdir}"
+
+make LIBINTL=NONE
+make LIBINTL=NONE DESTDIR=${prefix} prefix=/ install
+
+# Add .exe on Windows if it is missing
+if [[ -f ${bindir}/msgfmt ]]
+then
+    mv -n ${bindir}/msgfmt ${bindir}/msgfmt${exeext}
+    mv -n ${bindir}/msgmerge ${bindir}/msgmerge${exeext}
+fi
```

Comment:
Using gcc 6 or so would avoid this.

Reply:
I'm removing Windows builds since they are unlikely to work and I don't have a good way to test them.